src/updateChangelog.ts

```diff
@@ -174,11 +189,13 @@ async function updateChangelog({
       .getReleases()
       .find((release) => release.version === currentVersion)
   ) {
-    changelog.addRelease({ version: currentVersion });
+    // TODO: Get rid of typecast?
+    changelog.addRelease({ version: currentVersion as string });
   }
 
   if (isReleaseCandidate && hasUnreleasedChanges) {
-    changelog.migrateUnreleasedChangesToRelease(currentVersion);
+    // TODO: Get rid of typecast?
+    changelog.migrateUnreleasedChangesToRelease(currentVersion as string);
```

Comment:
`currentVersion` can be omitted, making theses cases somewhat ambiguous.

Reply:
Now that you'd added error handling above, this is guaranteed to be a string, despite `tsc` still not recognizing this.

So maybe these casts are fine? I don't know offhand how to reorganize this to avoid it, other than by duplicating that type guard.

For both of these casts, we'd might as well cast it to `Version` though.

Reply:
Done in f2b8390